documentation/src/main.css

```diff
@@ -16,4 +16,5 @@ body {
     height: 100%;
     overflow: hidden;
     margin: 0;
+    -webkit-font-smoothing: antialiased;
```

Comment:
should we also be applying the firefox version (`-moz-osx-font-smoothing`) of this? and/or is this something our postcss setup will handle?

Reply:
Great question! I didn't even know that property existed... At current, I've stolen this single property from the Spectrum CSS site. `-moz-osx-font-smoothing: grayscale;` seems to be the equivalent Firefox entry, but doesn't seem to apply a similar effect, which may be why they might have avoided this there.

I'll double check with the Spectrum CSS team about this.

Reply:
👍 we can move forward with this PR i guess and follow up with a second if there's a need for the firefox property.